tests/test_transform.py

```diff
@@ -1,13 +1,21 @@
 from __future__ import annotations
 
+# turbopuffer: Skip entire file - we use an optimized transform that bypasses Stainless
+# type-based transformations. Our _turbopuffer_transform only handles vector encoding
+# and Omit/NotGiven stripping, not field aliasing, date formatting, etc.
+# See _transform.py for details on why this optimization was needed (async performance).
+import pytest
+
+pytestmark = pytest.mark.skip(
+    reason="turbopuffer uses optimized transform that bypasses Stainless type-based transformations"
+)
```

Comment:
Can you compact this down to just

```
pytest.skip(
    reason="turbopuffer uses optimized transform that bypasses Stainless type-based transformations"
)
```

and move it to line 28 or so, so you don't have to move the `import pytest` up? With the way Stainless works, keeping these diffs to a minimum vastly decreases the chance of merge conflicts down the road.